src/client/styles/components/FilterPopup.scss

```diff
@@ -74,12 +74,8 @@
       // top: 50px;
       // z-index: 1000;
       display: block;
-      top: -240px;
       z-index: 2000;
-      left: -70px;
-      right: 200px;
-      width: 110%;
-      position: absolute;
```

Comment:
Right now, you cannot scroll on the popup window because it's fixed and you won't be able to get to the "apply button".

Try this and let me know what you think:
1) add back `position: absolute;` and add `top: -160px;` to this rule. You have to add the `-160px` so that the top looks decent but it's not perfect.
2) then remove the `border` and `box-shadow` from  the `.popup-container .popup` class rule, but leave it for `.popup-container .popup-no-js` class rule.

Just a suggestion so we can move on because it's looking much better than before, but just need to fix the scrolling.

Reply:
Let me try it then.

Reply:
I added height and overflow:scroll so it should fix the scroll issue.